### PR TITLE
Account popover

### PR DESCRIPTION
**Pull request by JakeSaterlay**
Adding a popover when you click the user avatar at the top of the screen, to navigate to your profile or to log out of the application

### CODE REVIEW

**Review by damiensedgwick — APPROVED** [review]
code looks good to me, just got a build failure to fix before merging 👌

File: packages/frontpage/app/(app)/layout.tsx [hunk 1]
```diff
@@ -86,15 +97,42 @@ export default async function Layout({
...
+          )}
+        </DropdownMenuTrigger>
+        <DropdownMenuContent className="w-56" side="bottom" align="end">
+          <form
+            action={async () => {
+              "use server";
+              await signOut();
+            }}
+          >
+            <DropdownMenuLabel>My Account</DropdownMenuLabel>
```
Comment (tom-sherman) [inline packages/frontpage/app/(app)/layout.tsx:117]: Show the handle here?

**Review by tom-sherman — APPROVED** [review]

File: packages/frontpage/app/(app)/layout.tsx (hunk 1 above)
Comment (damiensedgwick) [reply packages/frontpage/app/(app)/layout.tsx:117]: handles are too long imo and can be unwieldy on mobile.

File: packages/frontpage/app/(app)/layout.tsx [hunk 2]
```diff
@@ -86,15 +97,42 @@ export default async function Layout({
...
+            <DropdownMenuItem asChild>
+              <Link
+                href={`/profile/${session.user.name}`}
+                className="cursor-pointer"
+              >
+                Profile
+              </Link>
+            </DropdownMenuItem>
+            <DropdownMenuSeparator />
+            <DropdownMenuItem>
```
Comment (tom-sherman) [inline packages/frontpage/app/(app)/layout.tsx:128]:
```suggestion
            <DropdownMenuItem asChild>
```

I think you want `asChild` here. Currently there are nested interacted elements which is probs not accessible. Regardless, there's an extra dom element that doesn't need to be there.

**Review by tom-sherman — APPROVED** [review]

**Pull request merged**